k8s/contiv-vpp.yaml

```diff
@@ -6,7 +6,7 @@
 
 # This installs the contiv-etcd (ETCD server to be used by Contiv) on the master node in a Kubernetes cluster.
 # In odrer to dump the content of ETCD, you can use kubectl exec command similar to this:
```

Comment:
This installs contiv-etcd (the ETCD server to be used by Contiv) on the master node in a Kubernetes cluster.
  # In order to dump the content of ETCD, you can use the kubectl exec command similar to this: